### PR TITLE
fix: align search icon with dark mode icon

### PR DESCRIPTION
**Pull request by Julien-R44**
Sorry to create a PR for this but I still don't have write permission on this repo 😅

Before : 
image

After : 
image

### CODE REVIEW

**Comment by RomainLanz** [comment]
Does it work too with only `margin-block` instead of `margin`?

**Comment by Julien-R44** [comment]
Yep, works with `margin-block: auto` too if that's your question 
I didn't know this property!

File: resources/css/header.css [hunk 1]
```diff
@@ -72,6 +72,7 @@ HEADER STYLES
   width: var(--toc-width);
   padding: 0 var(--space-x) 0 10px;
   border-left: 1px solid var(--brand-mute-color);
+  margin: auto;
```
Comment (RomainLanz) [inline resources/css/header.css]:
```suggestion
  margin-block: auto;
```

**Review by RomainLanz — COMMENTED** [review]
Then let's use it! 😋

File: resources/css/header.css (hunk 1 above)
Comment (Julien-R44) [reply resources/css/header.css]: https://caniuse.com/?search=margin-block
Are you sure? This property seems to not be supported at all in IE, I think it won't concern any of our users, but what is the advantage of margin-block over simple margin in this case ( I'm going to read more about it on my side ) ?
Sorry to nitpick but I just want to understand 😅

Comment (Julien-R44) [reply resources/css/header.css]: Well anyway, I guess a lot of other things are broken on the site under IE. So let's gooo !

**Pull request merged**